serving/src/test/java/feast/serving/it/ServingServiceBigTableIT.java

```diff
@@ -399,6 +467,17 @@ private static void ingestData(
     return output.toByteArray();
   }
 
+  private static String trimAndHash(String expr, int maxLength) {
+    int maxPrefixLength = 40;
+    String finalName = expr;
+    if (expr.length() > maxLength) {
+      String hashSuffix =
+          Hashing.murmur3_32().hashBytes(expr.substring(maxPrefixLength).getBytes()).toString();
+      finalName = expr.substring(0, Math.min(expr.length(), maxLength)).concat(hashSuffix);
```

Comment:
so we cut original string to maxLength and then also adding suffix. And the final string length will be `maxLength + suffixLength`. I don't think this is correct